### ISSUE TITLE
US07 - Eu Como Usuário Desejo Saber Quais Métricas Usar para Classificação

### ISSUE THREAD

**Issue by Mexazonic** [fga-eps-mds/2019.1-PyLearner#130]
# US07 - Eu Como Usuário Desejo Saber Quais Métricas Usar para Classificação
### Descrição:
Nessa história de usuário o desenvolvedor deverá preparar o material para que o usuário do bot possa tirar dúvidas a respeito de métricas de classificação.
Os responsáveis deverão tratar de todas as tarefas que compõe essa história de forma sequencial, para que essa seja completa, abordando a maior parte dos modelos e tipos de problemas que este pode solucionar.

### Critérios de Aceitação
Para a aceitação desta história, a interação deve abordar as seguintes métricas:
- [x] Accuracy
- [x] Precision
- [x] Recall.
- [x] F1

Também torna-se necessário:
- [x] [Documentação de Estudo do tema](https://github.com/fga-eps-mds/2019.1-PyLearner/issues/)
- [x] [Intents, actions e templates Criados](https://github.com/fga-eps-mds/2019.1-PyLearner/issues/)
- [x] [Cenários/Histórias criadas](https://github.com/fga-eps-mds/2019.1-PyLearner/issues/)

**Comment by alexportof** [comment]
Encontramos alguns links de quais funções utilizar para cada métrica, [F1](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html#sklearn.metrics.f1_score), [Precision](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_score.html#sklearn.metrics.precision_score), [Recall](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html#sklearn.metrics.recall_score) e [Accuracy](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html#sklearn.metrics.accuracy_score) . O que acham?

**Comment by davialvb** [comment]
São essas mesmo! Podem usar tranquilo como referencia.

**Comment by alexportof** [comment]
[Documentação Finalizada!](https://github.com/fga-eps-mds/2019.1-PyLearner/blob/master/docs/documentation-mds/Documento-da-US07.md)

**Comment by Mexazonic** [comment]
Issue "Finalizada" e enviada para revisão!
Intents, actions e stories podem ser revisada por [aqui](https://github.com/pylearner-bot/pylearner-rasa/tree/130-US07-metrics/bot)
